src/registry/views/components-dependencies.pug

```diff
@@ -0,0 +1,9 @@
+#components-dependencies.box
+  each dependency in availableDependencies
+    .componentRow.row.table
+      a(href=dependency.link, target="_blank")
+        if dependency.core
+            p.release #{dependency.name} (node.js core dependency)
+        else
+            p.release #{dependency.name}@#{dependency.version}
+        
```

Comment:
Good catch @mattiaerre , what you think of adding prettier in the near future so that we don't have much nits anymore